account_tax_balance/readme/USAGE.rst

```diff
@@ -0,0 +1,5 @@
+Accounting --> Reporting --> Taxes Balance
+
+Select the company, the date range, the target moves and 'open taxes'
+
+.. figure:: https://user-images.githubusercontent.com/1336274/99388381-cd279300-28d5-11eb-9bb7-e8fe90d482af.png
```

Comment:
I'm afraid this change is not correct. It should be reverted to the previous one, with relative path, and build with the README generator in an absolute way, but only accessible once this is merged. Sorry for all the confusion around. If you think it's too much to do, we can merge it this way and then I'll do direct commit for reverting back.

Reply:
No problem. Done.

Reply:
I don't see the relative path, but I'm fixing after the merge.